packages/orbit-components/src/HorizontalScroll/index.tsx

```diff
@@ -157,22 +162,14 @@ const HorizontalScroll = React.forwardRef<HTMLDivElement, Props>(
           </>
         )}
         {arrows && (
-          <>
-            <ArrowButton
-              className="left-100"
-              isHidden={reachedStart || !isOverflowing}
-              onClick={() => handleClick("left")}
-            >
-              <ChevronBackward customColor={arrowColor} />
-            </ArrowButton>
-            <ArrowButton
-              className="right-100"
-              isHidden={reachedEnd || !isOverflowing}
-              onClick={() => handleClick("right")}
-            >
-              <ChevronForward customColor={arrowColor} />
-            </ArrowButton>
-          </>
+          <ArrowButton
+            className="left-100"
+            isHidden={reachedStart || !isOverflowing}
+            onClick={() => handleClick("left")}
+            title={arrowLeftTitle}
+          >
+            <ChevronBackward customColor={arrowColor} />
```

Comment:
It looks like we are not handling arrows properly in RTL, I will create ticket for it.

Reply:
You're right, the arrow icons are displayed wrongly.

<img width="549" alt="image" src="https://github.com/user-attachments/assets/7c53833d-c13a-47b1-82e9-36d1de5b28cb" />

The right arrow shouldn't be there, and the left one is missing 😬 

I'll create the RTL story, and it can be addressed separately, thanks!

Reply:
Resolving in favor of the Jira ticket created by Domi.

https://kiwicom.atlassian.net/browse/FEPLT-2235